specification/client/models/common.models.tsp

```diff
@@ -124,3 +126,22 @@ union DotNetChatVoiceIds {
 union DotNetRealtimeVoiceIds {
   VoiceIdsShared,
 }
+
+// CollectionQueryParameters
+
+alias CollectionLimitQueryParameter = {
+  /**
+   * A limit on the number of objects to be returned. Limit can range between 1 and 100, and the
+   * default is 20.
+   */
+  @query pageSizeLimit?: int32 = 20;
```

Comment:
Should this name match the REST API: `pageSizeLimit` vs. `limit` ?

Reply:
This custom model doesn't drive generation of the REST client, so I've named the property the way we want it for the client model. Does that address your concern?